src/main/java/uk/gov/hmcts/reform/civil/controllers/testingsupport/TestingSupportController.java

```diff
@@ -146,4 +148,22 @@ public ResponseEntity<String> getJudgeRevisitProcessEvent(@PathVariable("state")
         return new ResponseEntity<>(responseMsg, HttpStatus.OK);
     }
 
+    @GetMapping("/testing-support/unless-order/trigger-judge-revisit-process-event/{state}")
```

Comment:
where are you using this ? i cant see an api test for it in ga ccd or that needs to be added

Reply:
Hey Vasu, I have it to support api test.